schemas/rum/_common-schema.json

```diff
@@ -390,6 +395,21 @@
           "type": "string",
           "description": "SDK name (e.g. 'logs', 'rum', 'rum-slim', etc.)",
           "readOnly": true
+        },
+        "retry_count": {
```

Comment:
don't we send this information also in the request headers ? why do we need it also here ?